x-pack/plugins/infra/public/components/asset_details/tabs/overview/kpi_grid.tsx

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+
+import { EuiFlexGroup, EuiFlexItem, EuiSpacer } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { type KPIChartProps, Tile } from './tile';
+import { TOOLTIP } from './translations';
+import type { KPIProps } from './overview';
+import type { StringDateRange } from '../../types';
+
+const KPI_CHARTS: Array<Omit<KPIChartProps, 'loading' | 'subtitle'>> = [
```

Comment:
Would it be possible to reuse the config we have for the Hosts page? https://github.com/elastic/kibana/blob/main/x-pack/plugins/infra/public/pages/metrics/hosts/components/kpis/kpi_grid.tsx

Reply:
I moved the config to a common folder and now I use the same in both places 👍